src/interfaces/IERC20Permit.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+import { IERC20 } from './IERC20.sol';
+
+interface IERC20Permit is IERC20 {
```

Comment:
It's odd that `IERC20Permit` is `IERC20` while `IERC20Burn` isn't.

Reply:
I agree, will fix.